lib/model/maalcie/MaaltijdAanmeldingenModel.class.php

```diff
@@ -31,23 +30,24 @@ public function aanmeldenVoorMaaltijd($mid, $uid, $doorUid, $aantalGasten = 0, $
 			}
 		}
 
-		if ($this->getIsAangemeld($mid, $uid)) {
+		if ($this->getIsAangemeld($maaltijd->maaltijd_id, $uid)) {
 			if (!$beheer) {
 				throw new Exception('Al aangemeld');
 			}
 			// aanmelding van lid updaten met aantal gasten door beheerder
-			$aanmelding = $this->loadAanmelding($mid, $uid);
+			$aanmelding = $this->loadAanmelding($maaltijd->maaltijd_id, $uid);
 			$verschil = $aantalGasten - $aanmelding->aantal_gasten;
-			if ($verschil === 0) {
-				throw new Exception('Al aangemeld met ' . $aantalGasten . ' gasten');
-			}
+			// Kan nu even niets schelen
```

Comment:
Goeie, ik vind het vervelend als ie gaat mekkeren hierover. De ux van het toevoegen en verwijderen van eters is nog wel een puntje om eventueel naar te kijken.